Editor/Deployment/DeploymentStatusPoller.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Threading.Tasks;
+using AmazonGameLiftPlugin.Core.DeploymentManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared;
+
+namespace AmazonGameLift.Editor
+{
+    internal class DeploymentStatusPoller
+    {
+        public event Action InfoUpdated;
```

Comment:
nit: I typically use this casing only for constants. Rename to `infoUpdated` unless the whole codebase is using the same covention.

Reply:
The rest of the codebase is using Pascal for events